{"name": "test_login_3[1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_fixture.TestLogin object at 0x000001AA3759B240>, data = 1\n\n    def test_login_3(self, data):\n        \"\"\"记住密码登录\"\"\"\n        allure.attach(\"输入用户名\", \"输入用户名username_3\")\n        print(\"输入用户名\")\n        allure.attach(\"输入密码\", \"输入用户名password_3\")\n        print(\"输入密码\")\n        allure.attach(\"点击登录\", \"登录成功\")\n        print(\"点击登录\")\n        print(data)\n>       assert False\nE       assert False\n\nscripts\\test_fixture.py:40: AssertionError"}, "description": "记住密码登录", "attachments": [{"name": "输入用户名username_3", "source": "75bf2e8a-e635-4912-952f-60a680b0f079-attachment.attach"}, {"name": "输入用户名password_3", "source": "36754f72-1f1e-44ee-ad85-7baec46af65d-attachment.attach"}, {"name": "登录成功", "source": "0a72203c-2c52-49d4-8809-5bb0a8319e70-attachment.attach"}], "parameters": [{"name": "data", "value": "1"}], "start": 1575025910653, "stop": 1575025910660, "uuid": "88ab1d56-1a28-4fe6-ab8b-be7e1d031683", "historyId": "71b61181cb38681bfc85e48c867026dd", "testCaseId": "133a8e3ccd283ba22e290157fe855949", "fullName": "scripts.test_fixture.TestLogin#test_login_3", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47A72RD"}, {"name": "thread", "value": "8984-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture"}]}